src/table_plane_extractor_srv.py

```diff
@@ -5,12 +5,26 @@
 import numpy as np
 import rospy
 import open3d as o3d
+import compas.geometry.bbox as compas_bb
 from open3d_ros_helper import open3d_ros_helper as orh
 import tf2_ros
 from v4r_util.util import o3d_bb_to_ros_bb, transformPointCloud
 from vision_msgs.msg import BoundingBox3DArray
 
 
+def get_minimum_oriented_bounding_box(o3d_pc):
+    '''
+    Computes the oriented minimum bounding box of a set of points in 3D space.
+    Input: open3d.geometry.PointCloud o3d_pc
+    Output: open3d.geometry.OrientedBoundingBox o3d_bb
+    '''
+    bb_points = np.array(
+        compas_bb.oriented_bounding_box_numpy(np.array(o3d_pc.points)))
```

Comment:
use np.asarray instead of np.array to avoid copying data and reuse the same memory instead

Reply:
Updated